roles/create_mce_crs/tasks/main.yml

```diff
@@ -0,0 +1,72 @@
+- name: Create Template for Namespace for imported cluster
+  command: "oc apply -f {{ repo_root_path }}/roles/create_mce_crs/files/01_create-ns-template.yml --kubeconfig {{kubeconfig_dest_dir}}{{kubeconfig_dest_filename}}"
+
+- name: Create Template for Pull Secret
+  command: "oc apply -f {{ repo_root_path }}/roles/create_mce_crs/files/02_create-pull-secet-template.yml --kubeconfig {{kubeconfig_dest_dir}}{{kubeconfig_dest_filename}}"
+
+- name: Create Template for InfraEnv
+  command: "oc apply -f {{ repo_root_path }}/roles/create_mce_crs/files/03_create-infraenv-template.yml --kubeconfig {{kubeconfig_dest_dir}}{{kubeconfig_dest_filename}}"
+
+- name: Create Template for Cluster Image Set
+  command: "oc apply -f {{ repo_root_path }}/roles/create_mce_crs/files/04_create-cluster-image-set-template.yml --kubeconfig {{kubeconfig_dest_dir}}{{kubeconfig_dest_filename}}"
+
+- name: Create Template for Agent Cluster Install
+  command: "oc apply -f {{ repo_root_path }}/roles/create_mce_crs/files/05_create-agent-cluster-install-template.yml --kubeconfig {{kubeconfig_dest_dir}}{{kubeconfig_dest_filename}}"
+
+- name: Create Template for Import Cluster (ClusterDeployment)
+  command: "oc apply -f {{ repo_root_path }}/roles/create_mce_crs/files/07_create-cluster-deployment-template.yml --kubeconfig {{kubeconfig_dest_dir}}{{kubeconfig_dest_filename}}"
+
+- name: Create Template for Managed Cluster
+  command: "oc apply -f {{ repo_root_path }}/roles/create_mce_crs/files/managed-cluster-template.yml --kubeconfig {{kubeconfig_dest_dir}}{{kubeconfig_dest_filename}}"
+
+- name: Create Namespace for imported cluster
+  shell: "oc process create-ns-template -p NAMESPACE={{ namespace }} | oc create -f -"

```

Comment:
I don't see where the value for `namespace` is defined. If its user input then can you add a check for it.

Reply:
It is an user input in "inventory.yml". What kind of check you mean?

Reply:
Add a check to validate inventory perhaps with a var `use_mce_for_day2` and then validate_inventory to the meta of the role.